product_docs/docs/biganimal/release/known_issues/known_issues_dha.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Known Issues with Distributed High Availability/PGD
+navTitle: Distributed High Availability/PGD Known Issues
+deepToC: true
+---
+
+These are currently known issues in EDB Postgres Distributed (PGD) on BigAnimal as deployed in Distributed High Availability clusters. These known issues are tracked in our ticketing system and are expected to be resolved in a future release. 
+
+## Management/Administration
+
+### Deleting a PGD Data Group may not fully reconcile 
+When deleting a PGD Data Group, the target group resources will be physically deleted, but in some cases we have observed that the PGD nodes may not be completely partitioned from the remaining PGD Groups.  It’s recommended to avoid use of this feature until this is fixed and removed from the known issues list. 
+
+### Adjusting PGD cluster architecture may not fully reconcile
+In rare cases, we have observed that changing the node architecture of an existing PGD cluster may not complete. If a change has not taken effect in 1 hour, reach out to Support. 
+
+### PGD Cluster may fail to create due to Azure SKU issue
+
+In some cases, although a regional quota check may have passed initially, when the PGD cluster is created, it may fail due a SKU critical for the Witness Nodes. 
```

Comment:
In some cases, although a regional quota check may have passed initially, when the PGD cluster is created provisioning may fail if SKU critical for the Witness Nodes is not available across three availability zones.

Reply:
@aaronsonntag @djw-m 

Correction to your correction Aaron :)
"In some cases, although a regional quota check may have passed initially, when the PGD cluster is created provisioning may fail if a critical SKU for the Witness Nodes is not available across three availability zones.

Reply:
This is already merged so pushing in a separate commit.